### PR TITLE
clean up intrinsic

### PR DESCRIPTION
**Pull request by thewilsonator**
Clean up the bits leftover from the refactor.

### CODE REVIEW

**Review by WalterBright — APPROVED** [review]

File: src/dmd/toir.d [hunk 1]
```diff
@@ -490,15 +490,12 @@ int intrinsic_op(FuncDeclaration fd)
 Lva_start:
     if (global.params.is64bit &&
         fd.toParent().isTemplateInstance() &&
-        fd.ident == Id.va_start)
+        id3 == Id.va_start &&
+        id2 == Id.stdarg &&
+        id1 == Id.stdc &&
+        (*md.packages)[1] == Id.core)
```
Comment (rainers) [inline src/dmd/toir.d]: packages are not in reverse order, so this should be `id1 == Id.core && (*md.packages)[1] == Id.stdc`.

Comment (thewilsonator) [reply src/dmd/toir.d]: Ah, thanks!

**Comment by rainers** [comment]
BTW: this comment seems wrong, btc not handled:
https://github.com/dlang/dmd/blob/4525eb4aa2aa90e3c99be700cb3132b7746c62fb/src/dmd/toir.d#L473

**Comment by thewilsonator** [comment]
I'll do this in two parts.

**Pull request merged**